packages/clerk-js/src/ui/elements/OrganizationPreview.tsx

```diff
@@ -52,13 +52,15 @@ export const OrganizationPreview = (props: OrganizationPreviewProps) => {
       elementId={descriptors.organizationPreview.setId(elementId)}
       gap={3}
       align='center'
+      as='span'
```

Comment:
OrganizationPreview might not always be a child of a button, probably it's ok, i just wanted to clarify

Reply:
Yes, do you think there will be an issue with this being always a `span`?

Reply:
No issue here